src/components/contributions/contributions.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import EditIcon from './edit-icon.svg'
+
+const Contributions = ({ data }) => {
+  const commit = data.github.repository.object.history.nodes[0]
+  const file = data.github.repository.object.file
+  const lastUpdated = new Date(commit.author.date).toLocaleString('en-AU', {
+    dateStyle: 'long',
+  })
+  const editLink = `https://github.com/NUbots/NUbook/edit/main/${file.path}`
+  return (
+    <div className='text-sm text-secondary dark:text-secondary-inverted mb-6 border-t-2 border-b-2 border-gray-200 dark:border-gray-800 py-3 flex'>
+      <div className='flex-grow'></div>
+      <div className='pr-3 capitalize'>
+        <a
```

Comment:
Because only the date is coloured, I think it's best to move this hyperlink so that it only wraps the date, not the words `Last updated`

Reply:
Ok sounds good. This is also the case for the edit icon and text however the icon is grey and 'edit' is not shown on mobile. Would it make sense to also change these for consistency? Maybe make the icon coloured on mobile?

Reply:
I think the functionality of the edit is fine as is, we have other logos which are grey and also links in NUbook but I think semantically just having the date as the actual hyperlink for the updated will be more intuitive